spec/directions/bake_references/v3_spec.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe Kitchen::Directions::BakeReferences do
+
+  let(:book1) do
+    book_containing(html:
+    <<~HTML
+      <div data-type="chapter">
+        <h1 data-type="document-title" id="chapTitle1">
+          <span class="os-part-text">Chapter </span>
+          <span class="os-number">1</span>
+          <span class="os-divider"> </span>
+          <span class="os-text" data-type="" itemprop="">Title Text Chapter 1</span>
+        </h1>
+        <div data-type="page">
+          #{metadata_element}
+          <div data-type="document-title" id="someId1">1.1 Page</div>
+          <p>
+            <a href="#auto_12345" data-type="cite">
+              <div data-type="note" class="reference" display="inline" id="auto_12345">
+                <h3 data-type="title">Reference 1</h3>
+              </div>
+            </a>
+            <a href="#auto_54321" data-type="cite">
```

Comment:
I'm not sure here :thinking: Is V3 is created for Anatomy? If yes, there are no notes there. We are moving sections from modules to EOB:

![image](https://user-images.githubusercontent.com/40228854/136030771-539968f2-801d-4ff6-ba88-032b4b0faeb8.png)